grid2op/tests/test_alert.py

```diff
@@ -13,45 +13,1234 @@
+        """
+        When 2 lines are attacked at different steps 2 and 3 and we raise 1 alert on the first attack
+        we expect a reward of -4 at step 3 
+        """
+        kwargs_opponent = dict(lines_attacked=[ATTACKED_LINE]+['48_53_141'], 
+                                   duration=[1,1], 
+                                   steps_attack=[2, 3])
+        with make(self.env_nm, test=True, difficulty="1", 
+                            opponent_attack_cooldown=0, 
+                            opponent_attack_duration=99999, 
+                            opponent_budget_per_ts=1000, 
+                            opponent_init_budget=10000., 
+                            opponent_action_class=PlayableAction, 
+                            opponent_class=TestOpponentMultiLines, 
+                            kwargs_opponent=kwargs_opponent,
+                            reward_class=AlertReward(reward_end_episode_bonus=42)
+            ) as env : 
+            env.seed(0)
+            env.reset()
+            for i in range(env.max_episode_duration()):
+                act = self.do_nothing
+                if i == 1 :
+                    act = self.env.action_space({"raise_alert": [0]})
+                elif i == 3 : 
+                    act = self.blackout_action
+                obs, reward, done, info = env.step(act)
+
+                if i in [2,3] : 
+                    assert env._oppSpace.last_attack is not None
+                elif i == 3 : 
+                    assert reward == -4
+                elif i == env.max_episode_duration()-1: 
+                        assert reward == 42
+                else : 
+                    assert reward == 0
+            
+                if done : 
+                    break
+
+# return -4
+    def test_assistant_reward_value_blackout_2_lines_attacked_different_step_in_window_only_1_alert_on_second_attacked_line(self) -> None:
+        """
+        When 2 lines are attacked at different steps 2 and 3 and we raise 1 alert on the second attack
+        we expect a reward of -4 at step 3 
+        """
+        kwargs_opponent = dict(lines_attacked=[ATTACKED_LINE]+['48_53_141'], 
+                                   duration=[1,1], 
+                                   steps_attack=[2, 3])
+        with make(self.env_nm, test=True, difficulty="1", 
+                            opponent_attack_cooldown=0, 
+                            opponent_attack_duration=99999, 
+                            opponent_budget_per_ts=1000, 
+                            opponent_init_budget=10000., 
+                            opponent_action_class=PlayableAction, 
+                            opponent_class=TestOpponentMultiLines, 
+                            kwargs_opponent=kwargs_opponent,
+                            reward_class=AlertReward(reward_end_episode_bonus=42)
+            ) as env : 
+            env.seed(0)
+            env.reset()
+            for i in range(env.max_episode_duration()):
```

Comment:
can you remove this "for" loop and make the thing sequential (you only need to do 3 actions, 3 steps and 3 checks, it's clearer without the for loop I think)